libpromises/evalfunction.c

```diff
@@ -1212,6 +1212,15 @@ static FnCallResult FnCallConcat(EvalContext *ctx, ARG_UNUSED const Policy *poli
 
 /*********************************************************************/
 
+static FnCallResult FnCallTrim(ARG_UNUSED EvalContext *ctx, ARG_UNUSED const Policy *policy, const FnCall *fp, const Rlist *finalargs)
```

Comment:
Error from travis:

```
evalfunction.c:1215:108: error: unused parameter ‘fp’ [-Werror=unused-parameter]
```

You need to add `ARG_UNUSED`.

Reply:
Thank you.